src/reducers/postReducer.js

```diff
@@ -0,0 +1,22 @@
+import post from '../data/post';
+import { ADD_POST, DELETE_POST, UPADTE_POST, UPDATE_POST } from '../actions/blogActions';
+
+const initialState = [{
+
+}];
+
```

Comment:
Instead of having an empty state like this you could just initiate state in the same like you define the reducer like this: 
```js
export default function reducer(state = [], action) {
```